sbomify/apps/core/tests/e2e/utils.py

```diff
@@ -42,7 +42,7 @@ def diff_images(
 def assert_screenshot(
     baseline_image_path: str | Path,
     current_image_path: str | Path,
-    threshold: float = 0.0,
+    threshold: float = 0.005,
```

Comment:
@rusko124 your thoughts?
All major visual testing tools (Percy, Chromatic, Playwright's built-in) use thresholds because pixel-perfect matching is mathematically impossible with lossy formats and browser rendering.